cves/CVE-2014-0050.yml

```diff
@@ -141,7 +155,9 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    Between September of 2013 and February of 2014, this vulnerability existed. Multiple "infinite loop" bugs were mentioned
+    in the commit history between those dates, but none of them addressed the bug at hand.
```

Comment:
Do you think these "infinite loops" were connected or unrelated? This might be worth mentioning.